backbone.js

```diff
@@ -1079,10 +1079,26 @@
       if (!wait) this.add(model, options);
       var collection = this;
       var success = options.success;
+      var forwardPristineError;
```

Comment:
... I can simply initialize `forwardPristineError` with a bogus value, so that removing it as an `'error'` listener is never problematic.

```suggestion
      var forwardPristineError = _.noop;
```